scripts/controller.js

```diff
@@ -0,0 +1,37 @@
+export class Controller {
+
+  constructor(stores, events) {
+    this.dataStore = stores.store;
+    this.events = events;
+
+    this.events.on('action:create', this.onCreate.bind(this));
+    this.events.on('action:update', this.onUpdate.bind(this));
+    this.events.on('action:delete', this.onDelete.bind(this));
+    this.events.on('action:sync', this.onSync.bind(this));
```

Comment:
I feel like we could map event names to `dataStore` method calls, though that'd be probably a little too much magic…